module/VuFind/src/VuFind/Controller/OAuth2Controller.php

```diff
@@ -176,7 +176,7 @@ public function __construct(
         $this->csrf = $csrf;
         $this->session = $session;
         $this->identityRepository = $ir;
-        $this->accessTokenTable = $at;
+        $this->accessTokenService = $at;
```

Comment:
While we're changing this code anyway, might it be a good opportunity to introduce [constructor property promotion](https://php.watch/versions/8.0/constructor-property-promotion) and eliminate a large amount of boilerplate code?

Reply:
Callables are not supported, but changed all properties.

Reply:
Interesting, I was not aware of that limitation, but this is nonetheless a step in the right direction! :-)